src/commands/playground/api.rs

```diff
@@ -30,11 +31,16 @@ pub struct PlaygroundRequest<'a> {
 #[derive(Debug, Serialize)]
```

Comment:
```suggestion
#[derive(Debug, Serialize)]
#[serde(rename_all = "camelCase")]
```

Reply:
I'm renaming the field on line 34. That seemed more consistent with the crateType name on 25. Happy to change it to renameAll, but would have to remove the field rename I think.

Reply:
yes